activestorage/netcdf_to_zarr.py

```diff
@@ -9,6 +9,52 @@
 from activestorage.config import *
 from kerchunk.hdf import SingleHdf5ToZarr
 
+import time
+import h5py
+
+
+def _correct_compressor_and_filename(content, varname, bryan_bucket=False):
```

Comment:
@bnlawrence here is the fix for the new compressor in filters feature of Kerchunk